pyproject.toml

```diff
@@ -38,6 +39,7 @@ pytest-mock = "^1.10"
 pytest-pythonpath = "^0.7.3"
 flake8 = "^3.7"
 flake8-bugbear = "^19.3"
+bandit = "~1.5.1"
```

Comment:
If you could move this entry up to keep things in alphabetical order, for no other reason than to appease my OCD, I'd really appreciate it  😊